pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -75,4 +83,5 @@ metadata:
   name:  {{.clusterName}}
   namespace: {{.eksaSystemNamespace}}
 spec:
+  imageLookupFormat: {{.BaseRegistry}}/{{.OSDistro}}-{{.OSVersion}}-kube-{{.KubernetesVersion}}.raw.gz
```

Comment:
We will need to customize this further later, possibly make the format extension substitutable and use the imageLookupBaseRegistry field for this. Fine for now though, revisit later.